### PR TITLE
Removed non-digitalocean specific dependencies

### PR DESCRIPTION
**Pull request by joaocosta-codacy**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,19 +7,13 @@ ENV HELM_VERSION=v2.15.1
 ENV HELM_SSM_VERSION=1.0.2
 ENV KUBECTL_VERSION=v1.13.4
 ENV DOCTL_VERSION=1.18.0
```
Comment (h314to) [inline Dockerfile]: latest is [1.33.1](https://github.com/digitalocean/doctl/releases/tag/v1.33.1)

File: Dockerfile [hunk 2]
```diff
@@ -7,19 +7,13 @@ ENV HELM_VERSION=v2.15.1
 ENV HELM_SSM_VERSION=1.0.2
 ENV KUBECTL_VERSION=v1.13.4
```
Comment (h314to) [inline Dockerfile]: latest stable is [v1.16.2](https://github.com/kubernetes/kubernetes/releases/tag/v1.16.2)

File: Dockerfile [hunk 3]
```diff
@@ -7,19 +7,13 @@ ENV HELM_VERSION=v2.15.1
 ENV HELM_SSM_VERSION=1.0.2
```
Comment (h314to) [inline Dockerfile]: latest is [1.0.4](https://github.com/codacy/helm-ssm/tree/1.0.4)

File: Dockerfile [hunk 4]
```diff
@@ -31,12 +25,9 @@ RUN apk add --no-cache python3==${PYTHON3_VERSION} && \
     helm repo add codacy-stable https://charts.codacy.com/stable/ && \
     curl -Lo /usr/local/bin/kubectl "https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl" && \
     chmod +x /usr/local/bin/kubectl && \
-    curl -Lo /usr/local/bin/yq "https://github.com/mikefarah/yq/releases/download/2.4.0/yq_linux_amd64" && \
-    chmod +x /usr/local/bin/yq && \
     curl -sL https://github.com/digitalocean/doctl/releases/download/v${DOCTL_VERSION}/doctl-${DOCTL_VERSION}-linux-amd64.tar.gz | tar -xzv && \
     apk add libc6-compat && \
```
Comment (h314to) [inline Dockerfile]: you might want to use `--no-cache` to reduce docker images

**Review by h314to — APPROVED** [review]
Some minor nits, otherwise lgtm. In particular, it might be a good idea to keep the tools updated. You could do it in a subsequent PR.

File: Dockerfile (hunk 3 above)
Comment (joaocosta-codacy) [reply Dockerfile]: Odd. curl fails to get this release. 1.0.3 works fine though. :/ Using 1.0.3 for now

**Pull request merged**